### PR TITLE
update: info on new kotlin course added

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

File: docs/topics/learning-materials-overview.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ You can use the following materials and resources for learning Kotlin:
 * [Java to Kotlin migration guide: Strings](java-to-kotlin-idioms-strings.md) – learn how to perform typical tasks with strings in Java and Kotlin.
 * [Kotlin Koans](koans.md) – complete exercises to learn the Kotlin syntax. Each exercise is created as a failing unit test and your job is to make it pass.
 * [Kotlin by example](https://play.kotlinlang.org/byExample/overview) – review a set of small and simple annotated examples for the Kotlin syntax.
+* [Kotlin Basics track](/https://hyperskill.org/tracks/18/) – learn all the Kotlin essentials while creating applications step by step on JetBrains Academy.
```
Comment (koshachy) [inline docs/topics/learning-materials-overview.md:9]:
```suggestion
* [Kotlin Basics track](https://hyperskill.org/tracks/18/) – learn all the Kotlin essentials while creating applications step by step on JetBrains Academy.
```

File: docs/topics/home.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ Find a way to learn Kotlin that works for you. There are lots of learning materi
 
 * [Kotlin by example](https://play.kotlinlang.org/byExample/overview)
 * [Kotlin Koans](koans.md)
+* [Kotlin Basics track](/https://hyperskill.org/tracks/18/)
```
Comment (koshachy) [inline docs/topics/home.md:65]:
```suggestion
* [Kotlin Basics track on JetBrains Academy](https://hyperskill.org/tracks/18/)
```

File: docs/topics/faq.md [hunk 3]
```diff
@@ -128,10 +128,14 @@ on the [Books](books.md) page. For more books, see the community-maintained list
 
 ### Are there any online courses available for Kotlin?
 
-There are a few courses available for Kotlin, including a [Pluralsight Kotlin Course](https://www.pluralsight.com/courses/kotlin-getting-started) by Kevin Jones, 
-an [O’Reilly Course](https://shop.oreilly.com/product/0636920052982.do) by Hadi Hariri and an [Udemy Kotlin Course](https://petersommerhoff.com/dev/kotlin/kotlin-beginner-tutorial/) by Peter Sommerhoff.
+You can learn all the Kotlin essentials while creating working applications with the [Kotlin Basics track](https://hyperskill.org/tracks/18) on JetBrains Academy.
```
Comment (koshachy) [inline docs/topics/faq.md:131]: Has this sentence been proofread? 🤔
TBH, I don't like "while creating working" here

**Review by koshachy — APPROVED** [review]
Thanks,
Look through my comments.

File: docs/topics/faq.md (hunk 3 above)
Comment (danil-pavlov) [reply docs/topics/faq.md:131]: Yeap, checked this line (and other content) with proofreaders

**Pull request merged**